tutorials/high_performance_networking/README.md

```diff
@@ -0,0 +1,2171 @@
+- Infiniband (IB)
+
+To identify the current mode, run `ibstat` or `ibv_devinfo` and look for the `Link Layer` value.
+
+```bash
+ibv_devinfo
+```
+
+??? abstract "See an example output"
+
+    In the example below, the `mlx5_0` interface is in Ethernet mode, while the `mlx5_1` interface is in Infiniband mode. Do not pay attention to the `transport` value which is always `InfiniBand`.
+
+    ```sh
+    hca_id: mlx5_0
+            transport:                      InfiniBand (0)
+            fw_ver:                         28.38.1002
+            node_guid:                      48b0:2d03:00f4:07fb
+            sys_image_guid:                 48b0:2d03:00f4:07fb
+            vendor_id:                      0x02c9
+            vendor_part_id:                 4129
+            hw_ver:                         0x0
+            board_id:                       NVD0000000033
+            phys_port_cnt:                  1
+                    port:   1
+                            state:                  PORT_ACTIVE (4)
+                            max_mtu:                4096 (5)
+                            active_mtu:             4096 (5)
+                            sm_lid:                 0
+                            port_lid:               0
+                            port_lmc:               0x00
+                            link_layer:             Ethernet
+
+    hca_id: mlx5_1
+            transport:                      InfiniBand (0)
+            fw_ver:                         28.38.1002
+            node_guid:                      48b0:2d03:00f4:07fc
+            sys_image_guid:                 48b0:2d03:00f4:07fb
+            vendor_id:                      0x02c9
+            vendor_part_id:                 4129
+            hw_ver:                         0x0
+            board_id:                       NVD0000000033
+            phys_port_cnt:                  1
+                    port:   1
+                            state:                  PORT_ACTIVE (4)
+                            max_mtu:                4096 (5)
+                            active_mtu:             4096 (5)
+                            sm_lid:                 0
+                            port_lid:               0
+                            port_lmc:               0x00
+                            link_layer:             InfiniBand
+    ```
+
+**For Holoscan Networking, we want the NIC to use the ETH link layer.** To switch the link layer mode, there are two possible options:
+
+1. On IGX Orin developer kits, you can switch that setting through the BIOS: [see IGX Orin documentation](https://docs.nvidia.com/igx-orin/user-guide/latest/switch-network-link.html).
+2. On any system with a NVIDIA NIC (including the IGX Orin developer kits), you can run the commands below from a terminal:
+
+    1. Identify the PCI address of your NVIDIA NIC
+
+        === "ibdev2netdev"
+
+            ```bash
+            nic_pci=$(sudo ibdev2netdev -v | awk '{print $1}' | head -n1)
```

Comment:
If we wanted to we can simply leave the device name off mlxconfig and it will set all mellanox interfaces to ethernet.

Reply:
oh weird, I know that works on other commands. Thanks